app/pages/project/components/ProjectNavbar/components/ProjectTitle/ProjectTitle.jsx

```diff
@@ -13,43 +20,74 @@ const H1 = styled.h1`
   text-shadow: 0 ${pxToRem(2)} ${pxToRem(3)} rgba(0,0,0,0.5);
 `;
 
-const StyledLink = styled(IndexLink).attrs({
+export const StyledLink = styled(IndexLink).attrs({
   activeClassName: 'active'
 })`
-  text-decoration: none;
+  border-bottom: ${pxToRem(3)} solid transparent;
   color: white;
+  text-decoration: none;
+  white-space: nowrap;
 
   &:hover,
   &:focus {
     border-bottom: ${pxToRem(3)} solid white;
   }
 `;
 
-const CheckMarkWrapper = styled.span`
-  font-size: 0.4em;
+export const StyledCheckMarkWrapper = styled.span.attrs({
+  'aria-title': zooniverseApprovedTranslation,
+  role: 'img',
+  title: zooniverseApprovedTranslation
+})`
+  font-size: 0.75rem;
+  margin-left: 1rem;
 `;
 
-const CheckMark = styled.i`
-  color: #00979d;
+export const StyledCheckMark = styled.i`
+  color: ${theme('mode', {
+    light: zooTheme.colors.brand.default
+  })};
+  text-shadow: none;
 `;
 
+export const StyledUnderReview = styled.small`
```

Comment:
I'm not sure this is the right place for this comment, and this would be another edge case, but if it's a really short project name the "Under Review" gets wrapped strangely - https://styled-components-theme.pfe-preview.zooniverse.org/projects/markb-panoptes/calbug.

Reply:
nevermind, updated branch looks good, not sure if looking at old one or something else weird on my end.

Reply:
Hah you're caught me deploying to staging without pushing up changes here yet. I added `white-space: nowrap`

Reply:
I'll stop doing that so you can finish reviewing 😁